lib/ansible/module_utils/facts/hardware/aix.py

```diff
@@ -202,6 +204,31 @@ def get_mount_facts(self):
                                                       'time': '%s %s %s' % (fields[4], fields[5], fields[6])})
         return mount_facts
 
+    def get_device_facts(self):
+        device_facts = {}
+        device_facts['devices'] = {}
+
+        lsdev_cmd = self.module.get_bin_path('lsdev', True)
+        lsattr_cmd = self.module.get_bin_path('lsattr', True)
+        rc, out_lsdev, err = self.module.run_command(lsdev_cmd)
+
+        for line in out_lsdev.splitlines():
+            field = line.split()
+
+            device_attrs = {}
+            rc, out_lsattr, err = self.module.run_command("%s -El %s" % (lsattr_cmd, field[0]))
+            for attr in out_lsattr.splitlines():
+                attr_field = attr.split()
```

Comment:
Since attr_field is a list, attr_fields would be a better name

(and for the 'field' list above as well)